microsetta_private_api/db/patches/0139.sql

```diff
@@ -0,0 +1,28 @@
+-- May 13, 2024
+-- Create two tables for observation categories and observation errors list
+-- Add observations column to barcode_scans table
+CREATE TABLE observation_categories (
+    categories VARCHAR(255) PRIMARY KEY
+);
+
+INSERT INTO observation_categories (categories) VALUES ('Sample'), ('Swab'), ('Tube');
+
+CREATE TABLE observation_errors (
+    category VARCHAR(255),
+    error VARCHAR(255),
```

Comment:
```suggestion
    category VARCHAR(255) NOT NULL,
    observation VARCHAR(255) NOT NULL,
```

Along with the column name change and enforcing `NOT NULL` constraints, want a couple of keys on this table:
1) A primary key, either an int or UUID (e.g. `observation_id UUID PRIMARY KEY DEFAULT uuid_generate_v4()`)
2) A unique key on the combination of category+observation to ensure we don't end up with duplicate values down the line.